structures/model/node.py

```diff
@@ -35,6 +35,9 @@ def loads_count(self):
 
         :return: `int` number of loads
         """
+        if self.loads is None
```

Comment:
Hi Ben, there seems to by a typo (colon missing).

```suggestion
        if self.loads is None:
```

In any case, I could correctly run all the tests using Python 3.10.
What is the exact test file that fails for you? And, what version of Python and OS are you using?

Reply:
Python 3.8.8 and Windows 10.  I either already found that missing colon (because it is not missing in my code) or I did a sloppy job of writing the edit to make the pull request.  Also, I am working in the spyder IDE which is a bit different than your recommended Pycharm environment.  I think the structure_solution_test was where the issue came up, but I am not sure now.